src/index.html

```diff
@@ -17,6 +21,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
```

Comment:
Add class for header
```suggestion
    <header class="header">
```